src/server/services/db/postgres.ts

```diff
@@ -17,7 +17,14 @@ export function getPool() {
   if (pool) {
     return pool;
   }
+
   pool = new Pool(pgDbConfig);
+
+  pool.on('error', (err) => {
+    captureException(err);
+    process.exit(-1);
```

Comment:
-1 is geen exit code, ik denk dat je voor 1 (generieke bailout error) zoekt: https://tldp.org/LDP/abs/html/exitcodes.html . 1 zegt niet zoveel dus als je nog iets met meer betekenis zoekt bestaat er nog wel wat anders gewoon lekkah 1, denk ik
```suggestion
    process.exit(1);
```